docs/content/en/docs/fleet-manager/metric-plugin.md

```diff
@@ -47,11 +47,42 @@ kubectl port-forward svc/default-thanos-query 9090:9090 --address 0.0.0.0
     link="./image/thanos-ui.jpeg"
     >}}
 
+## Apply more monitor settings with Fleet Application
+
+Run following command to create a avalanche pod and ServiceMonitor in the fleet:
```

Comment:
What is `avalanche pod`?

Reply:
avalanche is a tool widely use in prometheus community for performance test, forgot to update the link.